### DISCUSSION TITLE
Making the Intellisense work well with twMerge and twJoin

### DISCUSSION THREAD

**Discussion by bradennapier** [dcastil/tailwind-merge#175] [Show and tell]
I am not sure if others have done similar but just figured I would point out since I am using this I put together a regex to properly use intellisense when using `twMerge` outside of a `className=` context.  Only thing that doesn't work is the prettier plugin.

https://github.com/tailwindlabs/tailwindcss/discussions/7554#discussioncomment-4440751

lmk if anyone has a better method of doing this but it works well from what i can tell! 

![image](https://user-images.githubusercontent.com/15365418/208287904-e8a122e0-2b5a-4403-9832-b0421ba4af49.png)

**Comment by dcastil** [comment]
Nice! I also use a similar config (only the last two items are specific to tailwind-merge):

```json
{
    "tailwindCSS.experimental.classRegex": [
        "\\/\\*\\s?tw:\\s?\\*\\/\\s?'(.*)'",
        "tw-start([\\w\\W]*?)tw-end",
        "[cC][lL][aA][sS][sS].+[\"'`](.*)[\"'`]",
        "[sS][tT][yY][lL][eE].+[\"'`](.*)[\"'`]",
        "twJoin\\([\\w\\W]*?\\)",
        "twMerge\\([\\w\\W]*?\\)",
    ],
}
```
